orchagent/fdborch.cpp

```diff
@@ -174,21 +174,32 @@ void FdbOrch::update(sai_fdb_event_t type, const sai_fdb_entry_t* entry, sai_obj
                 update.add = false;
                 itr++;
 
-                storeFdbEntryState(update);
+                if (SAI_NULL_OBJECT_ID != bridge_port_id)
+                {
+                    if (!m_portsOrch->getPortByBridgePortId(bridge_port_id, update.port))
+                    {
+                        SWSS_LOG_ERROR("Failed to get port by bridge port ID 0x%" PRIx64, bridge_port_id);
+                        return;
+                    }
+                    SWSS_LOG_DEBUG("FDB flush: mac: %s  port: %s bridge port OID: 0x%" PRIx64 ".",
```

Comment:
Depending on the number of entries, this could be huge. Please change log-level

Reply:
Hi Prince, thanks for the review. BTW, priority of DEBUG is the least, so, if the number of entries is huge, doesn't it make sense to put least priority LOG level? Please explain if you think otherwise.

Reply:
You are right, I might have looked at the previous logs